### PR TITLE
Interp

### PR DESCRIPTION
**Pull request by mahshidaln**
# PR Type
Feature

# Short Description
Add gradient-based interpretability methods under the `interp` module.

# Tests Added
None

### CODE REVIEW

File: odyssey/interp/grad_attribution.py
Comment (amrit110) [inline odyssey/interp/grad_attribution.py:1]: this seems like a script that you run and not a module to import. so this can be moved to the top level of the repo similar to pretrain.py and finetune.py. so like interpret.py or something like that

**Review by amrit110 — APPROVED** [review]
